tests/stale_price.rs

```diff
@@ -0,0 +1,94 @@
+#![cfg(feature = "test-bpf")] // This only runs on bpf
+
+use {
+    bytemuck::bytes_of,
+    pyth_client::{id, MAGIC, VERSION_2, instruction, PriceType, Price, AccountType, AccKey, Ema, PriceComp, PriceInfo, CorpAction, PriceStatus},
+    pyth_client::processor::process_instruction,
+    solana_program::instruction::Instruction,
+    solana_program_test::*,
+    solana_sdk::{signature::Signer, transaction::Transaction, transport::TransportError},
+};
+
+async fn test_instr(instr: Instruction) -> Result<(), TransportError> {
```

Comment:
this method is copy-pasted from the other test suite. Can you extract it out into a common utilities module that both tests can share?

Reply:
Moved into common.rs